pkg/sidecar/docker_network.go

```diff
@@ -140,3 +147,18 @@ func (dn *DockerNetwork) ConfigureNetwork(ctx context.Context, cfg *sdknw.Config
 
 	return nil
 }
+
+func (dn *DockerNetwork) handleRoutingPolicy(ctx context.Context, policy sdknw.RoutingPolicyType) error {
+	var err *multierror.Error
+
+	for _, routing := range dn.externalRouting {
+		switch policy {
+		case sdknw.AllowAll:
+			err = multierror.Append(err, routing.enable(dn.nl))
+		default:
+			err = multierror.Append(err, routing.disable(dn.nl))
+		}
```

Comment:
I think it'd be nice to be explicit about the policy, and not have a default case - maybe add `DenyAll`, and always make sure one of the two values is set. WDYT?

Reply:
So I added the `DenyAll` rule. However, I left the default case because returning an error instead would make all other tests fail since none of them is setting the option. I think for now we should just let `DenyAll` as default as it was before to avoid breaking test cases compatibility.